opentech/apply/projects/views.py

```diff
@@ -49,3 +50,20 @@ class ApplicantProjectDetailView(DetailView):
 class ProjectDetailView(ViewDispatcher):
     admin_view = AdminProjectDetailView
     applicant_view = ApplicantProjectDetailView
+
+
+@method_decorator(staff_required, name='dispatch')
+class ProjectEditView(UpdateView):
+    form_class = ProjectEditForm
+    model = Project
+
+    def form_valid(self, form):
+        self.object = form.save(commit=False)
```

Comment:
I like to do the following to stay out the way of inheritance.
```python      
if form.has_changed():
    form.instance.user_has_updated_details = True
return super().form_valid(form)
````
I honestly dont know which is the "correct" approach, but personally dislike re-writing the `form_valid` method this way. 

Alternatively should this be on the form save method?

Reply:
Fixed and rebased.